python/pyarrow/plasma.pyx

```diff
@@ -588,3 +606,66 @@ def connect(store_socket_name, manager_socket_name, int release_delay,
                               result.manager_socket_name,
                               release_delay, num_retries))
     return result
+
+def put(PlasmaClient client, value, object_id=None):
```

Comment:
You should be able to specify types here for `value` like `list value` (since that's the only thing supported for now), and `ObjectID object_id=None`.

Reply:
Is there any reason not to make these methods on the `PlasmaClient`?

Reply:
The reason why these are not methods on the PlasmaClient is that there is already a method get on there, which gets the underlying Buffers. If that's preferable we can rename that method to get_buffers and have get and put be methods of the PlasmaClient. What do you think?